AGPs/network-buget.md

```diff
@@ -0,0 +1,68 @@
+| Title                   | Author                 | Status    | Created    |
```

Comment:
```suggestion
---
AGP: N/A
Title: A simple Network Budget
Author: Louis Giraux (@LouisGrx)
Status: Stage IV
Track: Finance
Created: 2019-11-20
---
```

Reply:
Please use the AGP header format provided in the templates directory.

Reply:
Happy to do so. Wouldn't display properly so used another one